src/Okta.Sdk.Abstractions/UrlHelper.cs

```diff
@@ -0,0 +1,49 @@
+﻿// <copyright file="UrlHelper.cs" company="Okta, Inc">
+// Copyright (c) 2018 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+
+namespace Okta.Sdk.Abstractions
+{
+    /// <summary>
+    /// A utility class to manage URLs.
+    /// </summary>
+    public static class UrlHelper
+    {
+        /// <summary>
+        /// Gets the Okta domain given an issuer.
+        /// </summary>
+        /// <param name="issuer">The issuer, for example, "https://test-org.oktapreview.com/oauth2/default".</param>
+        /// <returns>The Okta domain, for example, "https://test-org.oktapreview.com".</returns>
+        public static string GetOktaDomain(string issuer)
```

Comment:
I'm worried this name will be misleading, I'd call this `GetOktaRootUrl` because this returns the root Url for the given issuer.  I think of a Uri/Url like this: 

`{Scheme}://{DomainAlsoKnownAsHost}`

I would expect a method called GetOktaDomain to return only the `domain` (aka host) portion of the Uri/Url.

Reply:
I agree `GetOktaRootUrl` is better, I'll change it.